db/migrate/20220712173556_create_comments.rb

```diff
@@ -0,0 +1,8 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] The comments table has a `text` column according to the ERD given to you. Kindly add it.

- [ ] User reference to the comments table is missing, you have post reference to comments but user reference to comments is not added, kindly create a migration and add. It might look like 👇 

```ruby
 def change
    add_reference :comments, :author, null: false, foreign_key: {to_table: :users}
  end
```